src/linear_algebra/multivariate_polynomial.lean

```diff
@@ -302,10 +302,25 @@ instance : module α (mv_polynomial σ α) := finsupp.to_module _ α
 instance C.is_ring_hom : is_ring_hom (C : α → mv_polynomial σ α) :=
 by apply is_ring_hom.of_semiring
 
+variables (σ a a')
 lemma C_sub : (C (a - a') : mv_polynomial σ α) = C a - C a' := is_ring_hom.map_sub _
 
 @[simp] lemma C_neg : (C (-a) : mv_polynomial σ α) = -C a := is_ring_hom.map_neg _
 
+variables {σ} (p)
+theorem C_mul' : mv_polynomial.C a * p = a • p :=
```

Comment:
```suggestion
@[simp] theorem C_mul' : mv_polynomial.C a * p = a • p :=
```

Reply:
The same lemma in data.polynomial is not tagged with `simp`, so it might be unsuitable to be a simp lemma.